floodviz/templates/index.html

```diff
@@ -10,40 +11,45 @@
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/map.css') }}" />
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/peakflow.css') }}" />
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/hydrograph.css') }}" />
-	<title>Flood Viz</title>
-	<script type="application/ld+json">
-		{{ linked_data | tojson | safe }}
-
-	</script>
+	<script type="application/ld+json"> {{ linked_data | tojson | safe }} </script>
+	<script> var FV = FV || {}; </script>
+	{% if config.GA_ID != "null" %}
+		<script> FV.GA_RUN = true; </script>
+	{% else %}
+		<script> FV.GA_RUN = false; </script>
+	{% endif %}
```

Comment:
Not a huge fan of how I had to implement this. But wrapping the jinja condition around the entire GA script would cause event errors later with figure interactions. I thought about moving the ga_send_event, but it is called directly from html which makes it required up top here. This at least avoids importing the ga_analytics script when it is not needed.